.github/workflows/build.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Set Build Matrix Output
         id: set-matrix
-        run: echo "::set-output name=matrix::`./build_configurations_helper.py print-ci-json`"
+        run: echo "::set-output name=matrix::`./scripts/build_configurations_helper.py print-ci-json`"
```

Comment:
added new scripts folder where we put all our scripts. i thought this was a good idea bc i added a new script. also in to the future i think we should have a install.sh